state-chain/pallets/cf-flip/src/lib.rs

```diff
@@ -441,8 +441,12 @@ impl<T: Config> Bonding for Bonder<T> {
 	type AccountId = T::AccountId;
 	type Amount = T::Balance;
 
-	fn update_bond(authority: &Self::AccountId, bond: Self::Amount) {
-		Account::<T>::mutate_exists(authority, |maybe_account| {
+	fn get_bond(account_id: &Self::AccountId) -> Self::Amount {
+		Account::<T>::get(account_id).bond
+	}
```

Comment:
We don't need to add this - the information is already accessible through the AccountInfo trait, which is part of the `Chainflip` base config trait.

`<T::FundingInfo as cf_traits::AccountInfo>::bond();`